src/main/kotlin/com/sourcegraph/cody/listeners/CodyDocumentListener.kt

```diff
@@ -12,17 +12,17 @@ import com.sourcegraph.cody.agent.protocol.CompletionItemParams
 import com.sourcegraph.cody.agent.protocol.ProtocolTextDocument
 import com.sourcegraph.cody.autocomplete.CodyAutocompleteManager
 import com.sourcegraph.cody.autocomplete.action.AcceptCodyAutocompleteAction
-import com.sourcegraph.cody.chat.CodeEditorFactory
 import com.sourcegraph.cody.telemetry.TelemetryV2
 import com.sourcegraph.cody.vscode.InlineCompletionTriggerKind
 import com.sourcegraph.utils.CodyEditorUtil
 
 class CodyDocumentListener(val project: Project) : BulkAwareDocumentListener {
 
+  // TODO: Looks like this functionality is broken after the migration to webview
```

Comment:
is it still broken or did you manage fix it? if it need more attention can you pls create an issue for that? 🙏

Reply:
It's still broken, fix will be a bit more involved, I linked an linear issue